example-nlp.py

```diff
@@ -0,0 +1,2 @@
+lexicon = dict(zip(pd.read_csv('Downloads/vader_lexicon.csv', encoding='cp437', header=None)[[0, 1]].values.T[0], pd.read_csv('Downloads/vader_lexicon.csv', encoding='cp437', header=None)[[0, 1]].values.T[1]))
```

Comment:
A shebang would be awesome here, if you intended this for future use from the command line. The standard thing to do, if you haven't used one yet, is to put `#!/usr/bin/env python` at the top of a standalone python script, and then make it executable by running `chmod +x  example-nlp.py`

Reply:
This line is somewhat hard to read with all the embedded function calls. Perhaps each one into it's own variable, for easier readability?